<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialLine</class>
 <widget class="QWidget" name="MaterialLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>67</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>170</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>170</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>170</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>170</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="12" rowspan="2">
    <widget class="QPushButton" name="btnMinusFault">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="2">
    <widget class="QLabel" name="expectedQty">
     <property name="text">
      <string>Quantité</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#626262;&quot;&gt;Qté attendue : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13" rowspan="2">
    <widget class="QSpinBox" name="faultQty">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9" rowspan="2">
    <widget class="QSpinBox" name="returnedQty">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="reference">
     <property name="text">
      <string>Réference</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="8" rowspan="2">
    <widget class="QPushButton" name="btnMinusReturned">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="name">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="10" rowspan="2">
    <widget class="QPushButton" name="btnPlusReturned">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="15" rowspan="2">
    <widget class="QPushButton" name="btnPlusFault">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QCommandLinkButton" name="btnArchive">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Archiver</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnPlusReturned</sender>
   <signal>clicked()</signal>
   <receiver>returnedQty</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMinusReturned</sender>
   <signal>clicked()</signal>
   <receiver>returnedQty</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMinusFault</sender>
   <signal>clicked()</signal>
   <receiver>faultQty</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>958</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlusFault</sender>
   <signal>clicked()</signal>
   <receiver>faultQty</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>997</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>955</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returnedQty</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MaterialLine</receiver>
   <slot>returnQtyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>faultQty</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MaterialLine</receiver>
   <slot>faultQtyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>964</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>963</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnArchive</sender>
   <signal>clicked()</signal>
   <receiver>MaterialLine</receiver>
   <slot>archiveLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>returnQtyChanged(int)</slot>
  <slot>faultQtyChanged(int)</slot>
  <slot>archiveLine()</slot>
 </slots>
</ui>
